manager/apis/app/v1beta1/blueprint_types.go

```diff
@@ -66,6 +66,10 @@ type BlueprintModule struct {
 	// as well as module status in the future.
 	// +optional
 	AssetIDs []string `json:"assetIds,omitempty"`
+
+	// External services that are required for functionality of the module, format of the string should be a valid URL
+	// +optional
+	ExternalServices []string `json:"externalServices,omitempty"`
```

Comment:
https://github.com/fybrik/fybrik/pull/2053 will add this information under Network.URLs of a module together with dataset URLs.

Reply:
OK, even better